packages/react/src/Grid/Grid.tsx

```diff
@@ -22,17 +22,77 @@ export type GridColumnNumbers = {
   wide: GridColumnNumber
 }
 
-interface GridComponent
-  extends ForwardRefExoticComponent<PropsWithChildren<HTMLAttributes<HTMLDivElement>> & RefAttributes<HTMLDivElement>> {
+type GridDensity = 'low' | 'high'
+
+type GridMarginSize = 'small' | 'medium' | 'large'
```

Comment:
We're setting padding, shouldn't the name reflect this?

Reply:
Hm, I'm not really convinced. `margin` means something in CSS-land, and it's not what we're setting here. It's confusing for developers I think to have a prop called `margin*` which doesn't set a CSS `margin`.

If `padding` is what we're setting, why not just call it 'padding' (for now at least)?

`paddingBottom`, `paddingVertical` etc. is a pretty good description of what it does, imo

Reply:
Okay.